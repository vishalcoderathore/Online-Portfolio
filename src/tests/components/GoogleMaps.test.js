import React from 'react';
import { shallow } from 'enzyme';
import MapWithAMakredInfoWindow from '../../components/GoogleMaps';

test('should render google maps correctly', () => {
    const wrapper = shallow(<MapWithAMakredInfoWindow
        googleMapURL="https://maps.googleapis.com/maps/api/js?key=AIzaSyBgM64gpvjKwX3JhihSyRs2pX9N8npMFzI&v=3.exp&libraries=geometry,drawing,places"
        loadingElement={<div ></div>}
        containerElement={<div className="maps--containerElement"></div>}
        mapElement={<div className="maps--mapElement"></div>}
    />);
    expect(wrapper).toMatchSnapshot();
});