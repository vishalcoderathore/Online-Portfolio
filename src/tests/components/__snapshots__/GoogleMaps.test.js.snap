// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render google maps correctly 1`] = `
<withScriptjs(withGoogleMap(Component))
  containerElement={
    <div
      className="maps--containerElement"
    />
  }
  googleMapURL="https://maps.googleapis.com/maps/api/js?key=AIzaSyBgM64gpvjKwX3JhihSyRs2pX9N8npMFzI&v=3.exp&libraries=geometry,drawing,places"
  isOpen={false}
  loadingElement={<div />}
  mapElement={
    <div
      className="maps--mapElement"
    />
  }
  onToggleOpen={[Function]}
/>
`;
